src/test/java/io/swagger/inflector/controllers/SwaggerOperationControllerTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2015 SmartBear Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.swagger.inflector.controllers;
+
+import com.google.common.collect.Maps;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class SwaggerOperationControllerTest {
+
+    @Test
+    public void testFilenameExtraction() throws Exception {
```

Comment:
Consider leaving out "test" in the method name. It's redundant since you have the @Test annotation, and in my experience choosing a name like "extractsFilenamesCorrectly" helps you think of the test as a specification, which in turn nudges you towards "test first".

However, if other test methods in this particular codebase have testXXX names, you may want to keep the name just for the sake of consistency.